### PR TITLE
Add Connector:5.6.0

### PR DESCRIPTION
**Pull request by martinbreu**
https://jira.verbis.dkfz.de/browse/GBAITC-788

### CODE REVIEW

**Comment by martinbreu** [comment]
> I would remove all build documentation. This project is about the deployment of ready Docker images from Docker hub. The manual installation part can say here but please also without the build process.

Which file and lines do you mean?

File: doc/Store.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+## Environment
+
+### Database
+
+The Open-Source database Postresql 9.6 is used. The database connection uses the connection pool of Tomcat. 
+
+This webapp needs schema '**samply**' in the database '**samply.store**' under user '**samply**' and password '**samply**' under port `5432`. 
+
+To change these settings during build, search for these values in the **src/pom.xml** and adapt to your needs.
```
Comment (alexanderkiel) [inline doc/Store.md]: nix during build...

File: doc/Store.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    ```
+
+
+### Tomcat
+
+Requirements:
+
+   - [Java 8](#java)
+
+1. Download and unzip Tomcat: http://mirror.funkfreundelandshut.de/apache/tomcat/tomcat-8/v8.5.38/bin/apache-tomcat-8.5.38.zip (eg. to /opt/tomcat-store)
```
Comment (alexanderkiel) [inline doc/Store.md]: is there an official link?

File: doc/Store.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+[Service]
+Type=forking
+
+Environment=JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64/jre
+Environment=CATALINA_PID=/opt/tomcat-store/temp/tomcat.pid
+Environment=CATALINA_HOME=/opt/tomcat-store
+Environment=CATALINA_BASE=/opt/tomcat-store
+Environment='CATALINA_OPTS=-Xms512M -Xmx1024M -server -XX:+UseParallelGC'
+Environment='JAVA_OPTS=-Djava.awt.headless=true -Djava.security.egd=file:/dev/./urandom'
```
Comment (alexanderkiel) [inline doc/Store.md]: Do we need this setting? java.security.egd

File: doc/Store.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+[Install]
+WantedBy=multi-user.target
+
+```
+
+
+
+​	Windows: 
+
+​		Follow installer: http://ftp.fau.de/apache/tomcat/tomcat-8/v8.5.38/bin/apache-tomcat-8.5.38.exe
```
Comment (alexanderkiel) [inline doc/Store.md]: same here official link?

File: doc/Connector.md [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+## Environment
+
+### Database
+
+The Open-Source database Postresql 9.6 is used. The database connection uses the connection pool of Tomcat. 
+
+This webapp needs schema '**samply**' in the database '**samply.connector**' under user '**samply**' and password '**samply**' under port `5432`. 
+
+To change these settings during build, search for these values in the **src/pom.xml** and adapt to your needs.
```
Comment (alexanderkiel) [inline doc/Connector.md]: no build please

File: doc/Connector.md [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+
+[Install]
+WantedBy=multi-user.target
+
+```
+
+
+​	Windows: 
+
+​		Follow installer: http://ftp.fau.de/apache/tomcat/tomcat-8/v8.5.38/bin/apache-tomcat-8.5.38.exe
```
Comment (alexanderkiel) [inline doc/Connector.md]: official link

File: doc/IMPORT.md [hunk 7]
```diff
@@ -1,12 +1,9 @@
-
 # Introduction
```
Comment (alexanderkiel) [inline doc/IMPORT.md]: I would remove that. The project is about deploying the bridgehead. Not about importing data. We need a separate GitBook Documentation for that.

Comment (martinbreu) [reply doc/IMPORT.md]: Let us talk about that.
I thought this repo should contain all information to run a bridgehead (deploy, import data, connect to sample locator, ...) And do you have in mind that nobody gonna read this IMPORT.md, and that we will delete it maybe next month.

File: doc/Store.md (hunk 2 above)
Comment (dolchchr) [reply doc/Store.md]: https://tomcat.apache.org/download-80.cgi

File: doc/Store.md (hunk 3 above)
Comment (martinbreu) [reply doc/Store.md]: This was a recomendation to speed up Tomcat in some cases, it worked so far and should also work without the`security.egd` , so should I really delete it?

**Pull request merged**